lib/ember-qunit/skip.js

```diff
@@ -0,0 +1,10 @@
+import testWrapper from 'ember-qunit/test-wrapper';
+import { skip as qunitSkip } from 'qunit';
+
+export default function skip(/* testName, expected, callback, async */) {
+  for (var _len = arguments.length, args = new Array(_len), _key = 0; _key < _len; ++_key) {
```

Comment:
Happy to make that change; I wrote it this way to keep it in line with e.g. [`only`].

[`only`]: https://github.com/rwjblue/ember-qunit/blob/master/lib/ember-qunit/only.js

I'd be happy to update `only` and `test` as well, if you'd like.